plugins/auth/index.js

```diff
@@ -158,88 +184,90 @@ exports.register = (server, options, next) => {
                     // Token is an API token
                     // using function syntax makes 'this' the request
                     const request = this;
-                    const tokenFactory = request.server.app.tokenFactory;
-                    const userFactory = request.server.app.userFactory;
-                    const pipelineFactory = request.server.app.pipelineFactory;
-                    const collectionFactory = request.server.app.collectionFactory;
+                    const { tokenFactory } = request.server.app;
+                    const { userFactory } = request.server.app;
+                    const { pipelineFactory } = request.server.app;
+                    const { collectionFactory } = request.server.app;
```

Comment:
these lines can kind of be combined
but we can do that later

Reply:
yea, it's not smart enough to combine lines.